apps/launch/common-util/constants/stakingContract.ts

```diff
@@ -102,3 +102,17 @@ export const STAKING_TOKEN_ADDRESSES: Addresses = {
 export const isSupportedChainId = (chainId: number): chainId is ChainId => {
   return chainId in IMPLEMENTATION_ADDRESSES;
 };
+
+type BlockNumber = {
+  [key in ChainId]: number;
+};
+
+export const blockNumbers: BlockNumber = {
+  [mainnet.id]: 20009952,
+  [optimism.id]: 0,
```

Comment:
0 is a valid block height, might be worth changing to null if it represents contract not being deployed or something similar